OracleManagementAgent/README.md

```diff
@@ -0,0 +1,118 @@
+    $ docker volume inspect mgmtagent-volume|grep Mountpoint
+        "Mountpoint": "/var/lib/docker/volumes/mgmtagent-volume/_data",
+    ```
+
+1. Copy the Install Key (input.rsp) into the shared Docker volume Mountpoint
+
+    ```shell
+    # create any necessary dirs
+    $ mkdir -p /var/lib/docker/volumes/mgmtagent-volume/_data/mgmtagent_secret
+    $ cp input.rsp /var/lib/docker/volumes/mgmtagent-volume/_data/mgmtagent_secret/
+    ```
+
+1. [Apply Prerequisites for Deploying Management Agents](https://docs.oracle.com/en-us/iaas/management-agents/doc/perform-prerequisites-deploying-management-agents.html)
+
+    **Note: Without the prerequisites applied the agent may not function as expected.**
+
+1. Use Docker to start a container
+
+    ```shell
+    $ docker run -d --name mgmtagent-container --hostname mgmtagent1 -v mgmtagent-volume:/opt/oracle:rw --restart unless-stopped oracle/mgmtagent-container:latest
+    ```
+
+    **Description of Docker run parameters used above**
+    <!-- markdownlint-disable MD033 -->
+    | Parameter | Description |
+    | --------- | ----------- |
+    | -d | Starts mgmtagent-container in detached mode |
+    | --name mgmtagent-container | The name given to the container to identify it. |
+    | --hostname mgmtagent1 | Assign mgmtagent1 as the containers internal hostname. This can be any hostname compliant string and it will be used to identify the Management Agent instance in the OMC Console. |
+    | -v mgmtagent-volume | /opt/oracle:rw: Mounts the volume mgmtagent-volume created on host filesystem inside the container at /opt/oracle with Read/Write privileges. |
+    | --restart unless-stopped | Unless explicitly stopped, this restart policy restarts mgmtagent-container automatically when docker restarts. |
+    <!-- markdownlint-enable MD033 -->
+
+
+#### Helpful Docker Administration commands
+
+1. Starting a stopped Management Agent Container
+
+    ```shell
+    $ docker container start mgmtagent-container
+    ```
+
+1. Stopping a running Management Agent Container
+
+    ```shell
+    $ docker container stop mgmtagent-container
+    ```
+
+1. Inspecting logs of Management Agent Container
+
+    ```shell
+    $ docker container logs mgmtagent-container
+    ```
+
+## License
+To download and run the Oracle Management Agent, regardless whether inside or outside a container, you must download the binaries from the Oracle website and accept the license indicated at that page.
+
+Oracle Linux is licensed under the [Oracle Linux End-User License Agreement](https://oss.oracle.com/ol/EULA).
+
+All scripts and files hosted in this project and GitHub [`docker-images/OracleManagementAgent`](./) repository, required to build the Docker images are, unless otherwise noted, released under the [UPL 1.0](https://oss.oracle.com/licenses/upl/) license.
+
+## Customer Support
+Oracle Management Agent container image is supported for Oracle Linux 7. For more details please see My Oracle Support.
```

Comment:
I'm not sure I understand this line. Can I run this container only on Oracle Linux 7? Please provide a link to the specific My Oracle Support Article ID.

Reply:
Updated the doc and provided a link that better describes supported OS for Management Agent